### DISCUSSION TITLE
need help with nvidia driver

### DISCUSSION THREAD

**Discussion by Dead-Slash** [prasanthrangan/hyprdots#1003] [Q&A]
hello everybody can any one help with this problem im wondering why the system take   GPU Driver : Hewlett-Packard Company HD Graphics 630 [103c:838f]  instead of an nvidia driver , can i get help im not too much familiar with the archlinux seeking for help i will join a screendhot of my neofetch
![Uploading 240302_00h10m27s_screenshot.png…]()

**Comment by Dead-Slash** [comment]
![240302_00h20m39s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/144330579/7bb2270d-c6a3-4269-9a78-dff7332c513a)

**Comment by Enigma1309** [comment]
See this:- https://github.com/lutris/docs/blob/master/InstallingDrivers.md
this may help to install drivers
